distribution/docs/src/main/resources/_contents/_developing/developing-intro-contents.adoc

```diff
@@ -0,0 +1,816 @@
+Within this feature, refer to any dependencies of the application as well as any features that should start automatically.
+Other features should have install set to `manual`.
+
+The following example demonstrates configuring features to be auto-started. The naming convention for this feature is typically "`application name`" + "`-app`," as shown.
+
+.Auto-start features
+[source,xml,linenums]
+----
+<feature name="catalog-app" install="auto">
+    <feature>platform-app</feature>
+    <feature>catalog-core</feature>
+    <feature>catalog-core-metricsplugin</feature>
+    <feature>catalog-core-sourcemetricsplugin</feature>
+    <feature>catalog-transformer-thumbnail</feature>
+</feature>
+----
+
+=== Developing ${branding} Applications
+
+The ${branding} applications are comprised of components, packaged as Karaf features, which are collections of OSGi bundles.
+These features can be installed/uninstalled using the ${admin-console} or ${command-console}.
+${branding} applications also consist of one or more OSGi bundles and, possibly, supplemental external files.
+These applications are packaged as Karaf KAR files for easy download and installation.
+These applications can be stored on a file system or a Maven repository.
+
+A KAR file is a Karaf-specific archive format (*K*araf *AR*chive).
+It is a jar file that contains a feature descriptor file and one or more OSGi bundle jar files.
+The feature descriptor file identifies the application's name, the set of bundles that need to be installed, and any dependencies on other features that may need to be installed.
+
+==== Describing Application Services
+
+Given the modular nature of OSGi, some applications perform operations on the services themselves.
+In order to present, identify, and manipulate the services, they need descriptive identifying information.
+Any service that implements the `Describable` interface in `org.codice.ddf.platform.services.common` will
+have an obligation to provide this information. The relevant fields are as follows:
+
+* ID: a unique identifier for the service
+* Title: the informal name for the service
+* Description: a short, human-consumable description of the service
+* Organization: the name of the organization that wrote the service
+* Version: the current version of the service (example: 1.0)
+
+The only field with stringent requirements is the ID field. Format should be `[*product*].[*component*]`
+such as `ddf.metacards` or `ddf.platform`; while the [*component*] within a [*product*] may simply be
+a module or bundle name, the [*product*] itself should be the unique name of the plug-in or integration
+that belongs to the organization provided. Note that `ddf` as a [*product*] is reserved for core features
+only and is not meant to be used during extension or integration.
+
+==== Creating a KAR File
+
+The recommended method for creating a KAR file is to use the `features-maven-plugin`, which has a `create-kar` goal.
```

Comment:
We don't use this plugin anymore. We now use the `org.apache.karaf.tooling:karaf-maven-plugin` plugin. The sample code will also need to be updated to reflect that.